freshmaker/lightblue.py

```diff
@@ -299,7 +309,7 @@ def resolve_commit(self):
         # Find the additional data for Container build in Koji.
         try:
             data = self.get_additional_data_from_koji(self.nvr)
-        except KojiLookupError as e:
+        except (KojiLookupError, ValidationError) as e:
```

Comment:
The `data["error"]` in this except clause is not correct for this type of missing extra repo error.
Can you have a separate except clause to add the corresponding error to `data`? The error message will be present as the failed reason for this build, so we want more meaningful error message, and better to define a new custom Exception
```
    except MissingInstallMediaRepoError as e:
        log.error(e)
        data = self._get_default_additional_data()
        data["error"] = str(e)
```